### DISCUSSION TITLE
Achievement suggestion - Reach level 100 or higher on all skills on Artie/on one character

### DISCUSSION THREAD

**Discussion by Udrakan** [FakeFishGames/Barotrauma#8111] [Ideas and suggestions]
Title.
Name suggestions?

**Comment by ghost** [comment]
Unfortunately, Artie is a mere mortal... "surprisingly mortal!" (c) Voland from "The Master and Margarita"
Since there are other means for hitting 5 in all skills then munchkinning it to 200, both in same character class (to which Artie belongs, fortunately), any character will do.
Since people tends to dig Ayn Rand references nowadays (for no apparent reason - there are too many, IMHO), I'd suggest "Atlas Emerged".
Looks fitting.
